### PR TITLE
feat: adding panels for OntapS3SVM object

### PR DESCRIPTION
**Pull request by Hardikl**
image

image

### CODE REVIEW

**Comment by Hardikl** [comment]
Added data in buckt1,
image

Dashboard values with data, bucket belongs to `osc` svm
image

image

File: grafana/dashboards/cmode/s3ObjectStorage.json [hunk 1]
```diff
@@ -910,6 +910,741 @@
...
+              "exemplar": false,
+              "expr": "avg by(datacenter,cluster,svm)(ontaps3_svm_delete_object_total{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\"}) and topk($TopResources, avg by(datacenter,cluster,svm) (avg_over_time(ontaps3_svm_delete_object_total{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\"}[3h])))",
+              "hide": false,
+              "interval": "",
+              "legendFormat": "{{svm}} - DELETE",
+              "range": true,
+              "refId": "D"
+            }
+          ],
+          "title": "Top $TopResources Object Total by Operations",
```
[the remark this answers is not in the document]
Comment (Hardikl) [reply grafana/dashboards/cmode/s3ObjectStorage.json:1278]: Total would be not needed, changed `object` to `SVM` accordingly.

File: grafana/dashboards/cmode/s3ObjectStorage.json [hunk 2]
```diff
@@ -910,6 +910,741 @@
...
+              "refId": "D"
+            }
+          ],
+          "title": "Top $TopResources Object by Rate",
+          "transformations": [],
+          "type": "timeseries"
+        },
+        {
+          "datasource": "${DS_PROMETHEUS}",
+          "description": "Average latency for GET/PUT/HEAD/DELETE object operations.",
```
[the remark this answers is not in the document]
Comment (Hardikl) [reply grafana/dashboards/cmode/s3ObjectStorage.json]: Checking...

File: integration/test/dashboard_json_test.go [hunk 3]
```diff
@@ -105,6 +105,7 @@ var excludeCounters = []string{
 	"ndmp_session_",
 	"metrocluster_check_",
 	"volume_top_clients_",
+  "ontaps3_svm_",
```
Comment (cgrinds) [inline integration/test/dashboard_json_test.go:108]: should be indented

File: grafana/dashboards/cmode/s3ObjectStorage.json [hunk 4]
```diff
@@ -101,14 +101,14 @@
...
-        "h": 2,
+        "h": 3,
         "w": 24,
         "x": 0,
         "y": 1
       },
       "id": 107,
       "options": {
-        "content": "This dashboard requires ONTAP 9.7+. One action is required to use this dashboard: <br>1. On Harvest, enable the Rest collector for OntapS3 template in your `harvest.yml` config for configuration metrics",
+        "content": "This dashboard requires ONTAP 9.7+. Three actions are required to use this dashboard: <br>1. If you are using the Rest collector, then you must enable the OntapS3 counters in `$HARVEST/conf/rest/default.yaml`. <br>2. If you are using the ZapiPerf collector, then you must enable the OntapS3SVM counters in `$HARVEST/conf/zapiperf/default.yaml`. <br>3. If you are using the RestPerf collector, then you must enable the OntapS3SVM counters in `$HARVEST/conf/restperf/default.yaml`. Please see https://github.com/NetApp/harvest/discussions/3137 for more details.",
```
Comment (cgrinds) [inline grafana/dashboards/cmode/s3ObjectStorage.json:111]: Might be easier to read if last sentence is on a seperate line

**Comment by cgrinds** [comment]
In `grafana/dashboards/cmode/s3ObjectStorage.json` might want to bump the version from 2 to 3

**Comment by Hardikl** [comment]
> In `grafana/dashboards/cmode/s3ObjectStorage.json` might want to bump the version from 2 to 3

Done.

**Review by cgrinds — APPROVED** [review]

**Pull request merged**